package.json

```diff
@@ -68,5 +70,15 @@
   "bugs": {
     "url": "https://github.com/devfolioco/react-otp-input/issues"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "**/*.{js,jsx}": [
+      "npx prettier --write"
```

Comment:
@nirmalhk7 Just one thing: Can you remove npx from here? We have prettier as a dev dependency already!

Reply:
Fixed.